tools/mage/deploy/deploy.go

```diff
@@ -591,3 +584,31 @@ func customResourceVersion() string {
 	// and also the same format used by the master stack.
 	return fmt.Sprintf("%s (%s)", util.Semver(), util.CommitSha())
 }
+
+// Takes a string  and returns the panther- prefixed, lowercased slice of words(strings) (separated by spaces).
+// e.g "oRg-ApI" -> []string{"panther-org-api"}
+// e.g "one two THREE" -> []string{"panther-one", "panther-two", "panther-three"}
+func PantherNames(setString string) []string {
+	set := strings.Fields(setString)
+	for index, entry := range set {
```

Comment:
Nit: I prefer `i` for indexes:

```suggestion
	for i, entry := range set {
```

Either way is fine, but it's common Go style to use short variable names for receivers, indexes, file handles, etc